src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/OffenderService.kt

```diff
@@ -712,7 +712,7 @@ class OffenderService(
   }
 }
 
-fun UserEntity.cas1LimitedAccessStrategy() = if (this.hasQualification(UserQualification.LAO)) {
+fun UserEntity.casLimitedAccessStrategy() = if (this.hasQualification(UserQualification.LAO)) {
```

Comment:
this strategy makes use of the LAO Qualification, which i believe only CAS1 uses. Therefore, i don't think this should be renamed or used by CAS3. Otherwise, we're mixing CASs and it becomes difficult to untangle/reason on in the future.

For CAS3, i believe you should instead be passing through 

`LimitedAccessStrategy.ReturnRestrictedIfLimitedAccess(user.deliusUsername)`

or, you could introduce a new function like the following and use that instead to make the code a bit more readible?

`fun UserEntity.casLimitedAccessStrategy() = LimitedAccessStrategy.ReturnRestrictedIfLimitedAccess(user.deliusUsername)`

Reply:
Change the code by create a new function but to be honest the code is a little bit confused and not easy to follow when we get offender details

Reply:
Any proposals for improvement are welcome. I thought the new function definition was reasonably straight forward (at least compared to all the deprecated ones)

```
  fun getPersonSummaryInfoResults(
    crns: Set<String>,
    limitedAccessStrategy: LimitedAccessStrategy,
  ): List<PersonSummaryInfoResult>
```